Build/Docker/DataContainer/DockerFile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
+
+ENV User="influx" \
+    Password="openXDA2024" \
+    Token="dqsZWTgF5NOCi-FZ_k79c9iOMMpbFYhIOcr4L03FeRGjZiC-VoakWjNrj20Pwni8PiRgoOv2PgYtGLwPXRo2dw==" \
```

Comment:
Not super familiar with docker, but would it be possible to obfuscate the passwords we have in here?

Reply:
these are placeholders. They should get overwritten once the container spins up.